inno/insecure-cookie.yaml

```
@@ -0,0 +1,24 @@
+rules:
+- id: insecure-cookie
+  patterns:
+  - pattern-either:
+    - pattern: r.set_cookie(..., secure=False)
+    - pattern: r.set_cookie(..., secure=False, ...)
+  message: |
```

```suggestion
  message: >-
```

To make sure the message is displayed correctly in CLI contexts, we use the `>-` YAML operator for messages